tasks/icons/_template.scss

```diff
@@ -1,13 +1,21 @@
 @charset 'utf-8';
 
+$version: '<%= _.join([
+    _.sample('0123456789'),
+    _.sample('0123456789'),
+    _.sample('0123456789'),
+    _.sample('0123456789'),
+    _.sample('0123456789'),
+    ], '' ) %>';
+
 @font-face {
     font-family: '<%= fontName %>';
-    src: url('<%= fontPath %><%= fontName %>.eot');
-    src: url('<%= fontPath %><%= fontName %>.eot?#iefix') format('eot')<% if (formats.indexOf('woff2') !== -1) { %>,
-        url('<%= fontPath %><%= fontName %>.woff2') format('woff2')<% } %>,
-        url('<%= fontPath %><%= fontName %>.woff') format('woff'),
-        url('<%= fontPath %><%= fontName %>.ttf') format('truetype')<% if (formats.indexOf('svg') !== -1) { %>,
-        url('<%= fontPath %><%= fontName %>.svg#<%= fontName %>') format('svg')<% } %>;
+    src: url('<%= fontPath %><%= fontName %>.eot?v#{$version}');
+    src: url('<%= fontPath %><%= fontName %>.eot?v#{$version}#iefix') format('eot')<% if (formats.indexOf('woff2') !== -1) { %>,
+        url('<%= fontPath %><%= fontName %>.woff2?v#{$version}') format('woff2')<% } %>,
+        url('<%= fontPath %><%= fontName %>.woff?v#{$version}') format('woff'),
+        url('<%= fontPath %><%= fontName %>.ttf?v#{$version}') format('truetype')<% if (formats.indexOf('svg') !== -1) { %>,
+        url('<%= fontPath %><%= fontName %>.svg?v#{$version}#<%= fontName %>') format('svg')<% } %>;
```

Comment:
I don't know why #<%= fontName %> is needed. But is the use two hashtags supported or may this cause problems?

Reply:
This won't generate a double hashtag in the output since the #{..} notation is the official way to add a variable to a string this will result in ```/bundles/clientwebsite/font/icon.svg?v03818#icon```